GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -477,6 +477,7 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
             var itemSelected = gitItem != null;
             var isFile = itemSelected && gitItem.ObjectType == GitObjectType.Blob;
             var isFolder = itemSelected && gitItem.ObjectType == GitObjectType.Tree;
+            var isFileOrFolder = isFile || isFolder;
```

Comment:
The items that are displayed are either files or folders, so, if you need this `File | Folder`, that means that the item should be always displayed.

Reply:
Does not include submodules, so relevant

Reply:
We have submodules as well. Does it matter?

Reply:
For instance, clean on Submodules has no effect, is FileOrFolder does not apply there